### DISCUSSION TITLE
Python renderer using pmtiles

### DISCUSSION THREAD

**Discussion by time4tea** [protomaps/PMTiles#301] [Show and tell]
Hi - not really an issue - but I thought you might be interested.

I'm just playing about with a "pure" python renderer using pmtiles.  Its not packaged or anything more than a toy at the moment, but it can render some pretty nice looking maps.

You can find it at https://github.com/time4tea/pptiles

Thanks for pmtiles - very nice project.

**Comment by bdon** [comment]
Moving to Discussions. Thanks for sharing!

**Reply by whshyaige** [reply]
I'm just playing about with a "pure" python renderer using pmtiles. Its not packaged or anything more than a toy at the moment, but it can render some pretty nice looking maps.

You can find it at https://github.com/time4tea/pptiles

Thanks for pmtiles - very nice project.
